src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalController.java

```diff
@@ -89,7 +89,7 @@ public ResponseEntity findUsersByOrganisation(@Pattern(regexp = ORGANISATION_IDE
                                                       @RequestParam(value = "page", required = false) Integer page,
                                                       @RequestParam(value = "size", required = false) Integer size) {
 
-        return searchUsersByOrganisation(organisationIdentifier, showDeleted, true, "", page, size);
+        return searchUsersByOrganisation(organisationIdentifier, showDeleted, "true", "", page, size);
     }

```

Comment:
move "true" and  "" to ProfessionalAPIConstants file

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/603/commits/a2eba9506c479fa47b703014a1b64d23efe869d6